src/api.js

```diff
@@ -5,25 +5,34 @@ const api = axios.create({
 });
 
 export const getArticles = (categoryName) => {
-    return api
-      .get("/articles", { params: { category_name: categoryName } })
-      .then(({ data }) => {
-        return data.articles;
-      });
-  };
+  return api
+    .get("/articles", { params: { category_name: categoryName } })
+    .then(({ data }) => {
+      return data.articles;
+    });
+};
 
 export const getArticleByID = (articleID) => {
-    return api
-      .get(`/articles/${articleID}`)
-      .then(({ data }) => {
-        return data.articles;
-      });
-    };
+  return api.get(`/articles/${articleID}`).then(({ data }) => {
+    return data.articles;
+  });
+};
 
 export const getComments = (articleID) => {
-      return api
-        .get(`/articles/${articleID}/comments`)
-        .then(({ data }) => {
-          return data.comments;
-        });
-      };
\ No newline at end of file
+  return api.get(`/articles/${articleID}/comments`).then(({ data }) => {
+    return data.comments;
+  });
+};
+
+export const upvoteArticle = (articleID) => {
+  return api.patch(`/articles/${articleID}`, { inc_votes: 1 }).then(() => {
+    console.log("patched");
```

Comment:
There's no need to keep these console logs in now. The patch should work without a `.then` I beleive.